src/recon.rs

```diff
@@ -4426,23 +4426,38 @@ pub(crate) unsafe fn rav1d_filter_sbrow_lr<BD: BitDepth>(
     let y = sby * f.sb_step * 4;
     let ss_ver = (f.cur.p.layout == Rav1dPixelLayout::I420) as c_int;
     let h = f.sr_cur.p.p.h + 127 & !127;
+    let pix_stride = f.sr_cur.p.stride.map(BD::pxstride);
```

Comment:
Could we name this `pxstride` to be consistent with `BD::pxstride`.  `pix` sounds a little too much like pic IMO.

Reply:
done